apps/admin-gui-e2e/src/e2e/group-management-group_observer.cy.js

```diff
@@ -0,0 +1,80 @@
+/// <reference types="cypress" />
+
+context('Actions', () => {
+  const dbGroupName = 'g-o-test-group';
+  const dbGroupMember = 'g-o-testgroupmember';
+  const dbGroupAdmin = 'g-o-testgroupadmin';
+  const dbVoName = 'g-o-test-vo';
+  const dbApplicationItemTextFieldName = 'g-o-input-test';
+  const dbExtsource = 'internal';
+
+  before(() => {
+    if (Cypress.env('BA_USERNAME_GROUP_MANAGER')) {
+      sessionStorage.setItem('baPrincipal', '{"name": "groupObserver"}');
+      sessionStorage.setItem('basicUsername', Cypress.env('BA_USERNAME_GROUP_OBSERVER'));
+      sessionStorage.setItem('basicPassword', Cypress.env('BA_PASSWORD_GROUP_OBSERVER'));
+      cy.visit('service-access');
+    }
+  });
+
+  beforeEach(() => {
+    cy.visit('home').get(`[data-cy=${dbGroupName}]`).click();
+  });
+
+  it('test get member', () => {
+    cy.get('[data-cy=members]')
+      .click()
+      .get(`[data-cy=${dbGroupMember}-firstName-td]`)
+      .should('exist')
+      .click();
```

Comment:
```suggestion
```

Reply:
reused existing member's detail element to check observer makes it to the detail page